src/openms/include/OpenMS/DATASTRUCTURES/QTCluster.h

```diff
@@ -114,10 +153,34 @@ namespace OpenMS
     /// Keep track of peptide IDs and use them for matching?
     bool use_IDs_;
 
+    /// Whether current cluster is valid
+    bool valid_;
+
+    /** 
+     * @brief Whether initial collection of all neighbors is needed
+     *
+     * This variable stores whether we need to collect all annotations first
+     * before we can decide upon the best set of cluster points. This is
+     * usually only necessary if the center point does not have an annotation
+     * but we want to use ids.
+     *
+    */
+    bool collect_annotations_;
+
+    /// Whether current cluster is accepting new elements or not (if true, no more new elements allowed)
+    bool finalized_;
+
+    /// x coordinate in the grid cell
+    Int x_coord_;
+
+    /// y coordinate in the grid cell
+    Int y_coord_;
+
     /**
      * @brief Set of annotations of the cluster
      *
-     * The set of peptide sequences that is compatible to the cluster center and results in the best cluster quality.
+     * The set of peptide sequences that is compatible to the cluster center
+     * and results in the best cluster quality.
      */
     std::set<AASequence> annotations_;
```

Comment:
yes, it would suffice. However I think we currently dont have a hash function for `AASequence` yet but it would make sense to add this